src/web.ts

```diff
@@ -120,6 +121,11 @@ export class IntercomWeb extends WebPlugin implements IntercomPlugin {
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async displayArticle(_article: IntercomArticle): Promise<void> {
+    throw new Error('[Intercom] displayArticle is not implemented on web');
+  }
+
```

Comment:
Maybe a no-op with a log error ? Or open the article in a new tab using faq link?

Reply:
IMO, better a noop (but throwing an error is better no ?)
And the handling of opening the URL in a new tab can be handled by the app directly

Reply:
Yes, it make sense 🙂

Reply:
Maybe you can use this : https://github.com/ionic-team/capacitor/blob/main/core/src/web-plugin.ts#L93-L99